Dockerfile.amd64

```diff
@@ -5,6 +5,9 @@ FROM jellyfin/jellyfin-server:${TARGET_RELEASE}-amd64 as server
 FROM jellyfin/jellyfin-web:${TARGET_RELEASE} as web
 FROM debian:buster-slim
 
+LABEL maintainer "Jellyfin Packaging Team - packaging@jellyfin.org"
+LABEL org.opencontainers.image.source https://github.com/jellyfin/jellyfin
```

Comment:
```suggestion
LABEL maintainer="Jellyfin Packaging Team - packaging@jellyfin.org" \
    org.opencontainers.image.source="https://github.com/jellyfin/jellyfin"
```

Reply:
making a note here since I [received feedback](https://github.com/jellyfin/jellyfin-vue/pull/795#discussion_r583261061) on another comment regarding this and stress this works either way.
This format is a matter of preference and opinion of what is cleaner/clearer to the eye :wink:

(same goes for the other LABEL comments of mine)